### PR TITLE
Compare Cloud SOAR to the Automation Service

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request creates a new article:
https://help.sumologic.com/docs/cloud-soar/compared-to-automation-service/

It also updates this section:
https://help.sumologic.com/docs/cse/automation/about-automation-service-and-cloud-siem/#differences-compared-to-cloud-soar

Asana ticket: "[Article comparing Cloud SOAR to Automation Service](https://app.asana.com/0/1183741177809740/1206498216348536/f)"

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

### CODE REVIEW

File: docs/cloud-soar/compared-to-automation-service.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+title: Cloud SOAR Compared to the Automation Service
+sidebar_label: Compared to Automation Service
+description: Compare the features of Cloud SOAR with the Automation Service.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Cloud SOAR is a full-featured security orchestration, automation, and response (SOAR) application. The [Automation Service](/docs/platform-services/automation-service/about-automation-service/) is a subset of automation capabilities adapted from Cloud SOAR that is available to the entire Sumo Logic log analytics platform. 
+
+The Automation Service only has Cloud SOAR’s  playbook-related features, including App Central and the Automation Bridge. Like the [Cloud SOAR action types](/docs/cloud-soar/automation/#action), the [Automation Service action types](/docs/platform-services/automation-service/automation-service-playbooks/#add-an-action-node-to-a-playbook) can perform automated responses to events, including  run containment actions and manual user interaction steps.
```
Comment (kimsauce) [inline docs/cloud-soar/compared-to-automation-service.md:12]:
```suggestion
The Automation Service only has Cloud SOAR’s playbook-related features, including App Central and the Automation Bridge. Like the [Cloud SOAR action types](/docs/cloud-soar/automation/#action), the [Automation Service action types](/docs/platform-services/automation-service/automation-service-playbooks/#add-an-action-node-to-a-playbook) can perform automated responses to events, including  run containment actions and manual user interaction steps. 
```

File: docs/cloud-soar/index.md [hunk 2]
```diff
@@ -21,6 +21,12 @@ Navigate the evolving threat landscape with our best-in-class security orchestra
   <p>Cloud SOAR Security ops supports multi-tenant architecture.</p>
   </div>
 </div>
+<div className="box smallbox card">
+  <div className="container">
+  <a href="/docs/cloud-soar/compared-to-automation-service"><img src={useBaseUrl('img/icons/security/SOC.png')} alt="icon" width="40"/><h4>Cloud SOAR Compared to the Automation Service</h4></a>
+  <p>Compare the features of Cloud SOAR with the Automation Service..</p>
```
Comment (kimsauce) [inline docs/cloud-soar/index.md:27]:
```suggestion
  <p>Compare the features of Cloud SOAR with the Automation Service.</p>
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
small fixes

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce!

**Comment by jpipkin1** [comment]
@kimsauce, could you approve this PR?

**Review by kimsauce — APPROVED** [review]

**Comment by jpipkin1** [comment]
Thanks, @kimsauce!

**Pull request merged**